app/views/admin/developers/_form.html.erb

```diff
@@ -0,0 +1,51 @@
+<%= simple_form_for @developer do |f| %>
+
+<div class="row">
+  <div class="col s12">
+    <%= f.input :name, label: 'Nome', autofocus: true %>
```

Comment:
Not use label, use I18n. Remind that simple form changes the attribute name, to show it, getting the name of the locales file.